index.js

```diff
@@ -240,11 +246,11 @@ function grype_render_rules(vulnerabilities) {
                     "Location: "+v.artifact.locations[0].path+"\n"+
                     //"Data Namespace: "+v.vulnerability.matched_by.matcher +"\n"+
                     "Data Namespace: "+ "unknown" + "\n"+
-                    "Link: ["+v.vulnerability.id+"]("+v.vulnerability.links[0]+")",
+                    `Link: ${link}`,
                     "markdown": "**Vulnerability "+v.vulnerability.id+"**\n"+
                     "| Severity | Package | Version | Fix Version | Type | Location | Data Namespace | Link |\n"+
                     "| --- | --- | --- | --- | --- | --- | --- | --- |\n"+
-                    "|"+v.vulnerability.severity+"|"+v.artifact.name+"|"+v.artifact.version+"|"+"unknown"+"|"+v.artifact.type+"|"+v.artifact.locations[0].path+"|"+"unknown"+"|["+v.vulnerability.id+"]("+v.vulnerability.links[0]+")|\n"
+                    "|"+v.vulnerability.severity+"|"+v.artifact.name+"|"+v.artifact.version+"|"+"unknown"+"|"+v.artifact.type+"|"+v.artifact.locations[0].path+"|"+"unknown"+"|"+link+"|\n"
```

Comment:
This whole line might read easier as a template, and with values separated out. For example:

```javascript
const { severity } = v.vulnerability;
const { name, version, type, locations } = v.artifact;
const { path } = locations[0];

// ...

`|${severity}|${name}|${version}|unknown|${type}|${path}|unknown|${link}|\n`
```

Reply:
I agree. I've started using templates as I carve out changes. This whole file needs to use more templates. Do note that I have captured that feedback and created https://github.com/anchore/scan-action/issues/51 to track it.